charts/ccsm-helm/README.md

```diff
@@ -102,6 +102,11 @@ Information about Zeebe you can find [here](https://docs.camunda.io/docs/compone
 | | `service.commandName` | Defines the name of the Command API endpoint, where the broker commands are sent to | `command` |
 | | `service.internalPort` | Defines the port of the Internal API endpoint, which is used for internal communication | `26502` |
 | | `service.internalName` | Defines the name of the Internal API endpoint, which is used for internal communication | `internal` |
+| | `service.extraPort` | Exposes 1 or more to the service | `[]` |
+| | `service.extraPort.name` | A friendly name for your Port | `` |
+| | `service.extraPort.protocol` | Defines the protocol to be exposed | `TCP` |
+| | `service.extraPort.port` | Defines a port to be exposed | `` |
+| | `service.extraPort.targetPort` | Defines a targetPort to be mapped to the port | `` |
```

Comment:
:wrench: If we had only .Values.extraServices then we wouldn't need to document all these values.